starter_code/src/movies.js

```diff
@@ -2,14 +2,93 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+
+
+function orderByYear(arr) {
+  let newArr = [...arr];
+  newArr.sort(function(a, b){ 
+    if (a.year - b.year !== 0) {
+      return a.year - b.year;
+      }
+    else  {
+      if (a.title > b.title) {return 1}
+      else if (a.title < b.title) {return -1}
+      else {return 0};
```

Comment:
return a.title.localeCompare(b.title) would resolve to the same result.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare